zuul.d/project-templates.yaml

```diff
@@ -370,63 +370,48 @@
         - ansible-test-network-integration-iosxr-netconf-python27-stable211:
```

Comment:
Sorry, just noticed. Now that all these jobs are voting, we also need to add them into the gate pipeline below. I would recommend making a yaml anchor now, like we do above in &ansible-collections-cisco-iosxr-units-jobs